src/components/Keyword/Mobile/AlertForm.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const AlertForm = () => {
+  return (
+    <MobileAlertForm>
+      <AlertContainer>
+        <AlertHeader>
+          <ImportantAlert>중요알림</ImportantAlert>
+          <GeneralAlert>일반알림</GeneralAlert>
+        </AlertHeader>
+        <AlarmInner>
+          <AlramContentLine>
+            <AlarmInnerMain>무음모드 알림</AlarmInnerMain>
+            <AlarmInnerOption>
+              <ToggleImage src="/asset/ToggleOff.svg" />
+            </AlarmInnerOption>
+          </AlramContentLine>
+          <AlramContentLine>
+            <AlarmInnerMain>진동모드 알림</AlarmInnerMain>
+            <AlarmInnerOption>
+              <ToggleImage src="/asset/ToggleOff.svg" />
+            </AlarmInnerOption>
+          </AlramContentLine>
+          <AlramContentLine>
+            <AlarmInnerMain>확인 버튼 누를 때까지 알림</AlarmInnerMain>
+            <AlarmInnerOption>
+              <ToggleImage src="/asset/ToggleOff.svg" />
+            </AlarmInnerOption>
+          </AlramContentLine>
+          <AlramContentLine>
+            <AlarmInnerMain>알림주기</AlarmInnerMain>
+            <AlarmInnerOption>
+              0시간
+              <ArrowImage src="/asset/chevron-right.svg" />
+            </AlarmInnerOption>
+          </AlramContentLine>
+        </AlarmInner>
+      </AlertContainer>
+      <ErrorText>알림은 앱에서만 설정할 수 있습니다.</ErrorText>
+    </MobileAlertForm>
+  );
+};
+
+export default AlertForm;
+
+const MobileAlertForm = styled.div`
+  width: 100%;
+  height: auto;
+  flex-direction: column;
+`;
+const AlertContainer = styled.div`
+  width: 100%;
+  height: 254px;
+  border: 1px solid ${(props) => props.theme.colors.yellow};
+`;
+
+const AlertHeader = styled.div`
+  display: flex;
+  border-bottom: 1px solid ${(props) => props.theme.colors.lightgray};
+`;
+const AlertHeaderContent = styled.div`
+  width: 50%;
+  height: 48px;
+  font-size: 14px;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;

```

Comment:
이 세가지 속성이 동시에 사용되는 컴포넌트가 꽤 많던데, theme에 mixin 형태로 추가하는건 어떨까요?

```suggestion
  ${(props) => props.theme.mixins.flexMixin()}

// theme.js
theme = {
  colors: { ... },
  mixins: {
    flexMixin: (align = 'center', justify = 'center') => `
	display:flex;
	align-items: ${align};
	justify-content: ${justify};`,
  }
}
```

Reply:
상당히 효율적이고 코드를 줄이는데 도움이 될 것 같습니다. 지금 당장하기에는 시간적으로 어려워, 다음 리팩토링 기간에 수정할 수 있는 부분은 수정해보겠습니다